plugin/src/main/java/org/opensearch/ml/processor/InferenceProcessorAttributes.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.ml.processor;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.Getter;
+
+class InferenceProcessorAttributes {
+    @Getter
+    private final List<Map<String, String>> inputMaps;
```

Comment:
Can we add a comment here to show an example of the key and value for the Map. Just to let reader better understand what is expected in the key and value, and why we need a List of Map here.

Reply:
+1, please add more comments

Reply:
added comments